.github/CHANGELOG.md

```diff
@@ -44,6 +44,9 @@
 * Reduce the number of shots in the PennyLane Python tests on CIs, from 20k to 10k.
   [(#1046)](https://github.com/PennyLaneAI/pennylane-lightning/pull/1046)
 
+* Capture execution via `dev.eval_jaxpr` can now be used with `jax.jit` and `jax.vmap`.
```

Comment:
Could you add this to the top of the list (line 9)?

Reply:
moved.